_infra/helm/locust/values.yaml

```diff
@@ -18,24 +18,24 @@ locust:
       security_user_name: admin
       security_user_password: secret
       test_respondent_password: password
-      test_respondents: 5
-      GOOGLE_CLOUD_PROJECT: "ras-rm-performance-tests"
-      GCS_BUCKET_NAME: "ras-rm-performance-results"
+      test_respondents: 10
+      GOOGLE_CLOUD_PROJECT: "ras-rm-performance-20220908"
+      GCS_BUCKET_NAME: "ras-rm-performance-20220908-locust"
 
   master:
+    replicas: 1
     environment:
-      LOCUST_USERS: 1000
-      LOCUST_SPAWN_RATE: 100
-      LOCUST_RUN_TIME: 5m
+      LOCUST_USERS: 10
+      LOCUST_SPAWN_RATE: 1
+      LOCUST_RUN_TIME: 35m
```

Comment:
We run the tests for 35 mins, but the Spinnaker pipeline will undeploy the Locust apps after 30 mins. That means we only get a single set of aggregated output results for the run. Prior to this pattern we would see multiple result sets output, one each time Locust restarted.